pandas/tests/test_errors.py

```diff
@@ -22,12 +22,12 @@
 def test_exception_importable(exc):
     from pandas import errors
 
-    e = getattr(errors, exc)
-    assert e is not None
+    err = getattr(errors, exc)
+    assert err is not None
 
     # check that we can raise on them
-    with pytest.raises(e):
-        raise e()
+    with pytest.raises(err, match=""):
```

Comment:
the match argument is treated as a regex, did you mean "^$", i.e. no message?

Reply:
Correct, will fix.

(Thank you)